### ISSUE TITLE
Supports Auto Linking in RN 0.60+?

### ISSUE THREAD

**Issue by ShuKTan** [joltup/rn-fetch-blob#491]
Does this library support auto linking in react native 0.60+ version?

I could not find anything about auto linking in docs.

**Comment by Hristijan95** [comment]
Yes, it looks like auto linking is supported for RN 0.60.+, I've just tried.

**Comment by pcowgill** [comment]
In that case, should we change the name of this issue to something about documenting this? Or close this issue?

**Comment by pcowgill** [comment]
Based on this other issue, it sounds like we shouldn't close it just yet https://github.com/joltup/rn-fetch-blob/issues/520

**Comment by dipo1** [comment]
Auto linking could mean 2 things.
Before react-native 0.60, auto-linking used to mean running the "react-native link" command.
After react-native 0.60 auto-linking now means not having to run the "react-native link" command.

I can confirm that the module does not support the auto-linking feature that comes with react-native 0.60. At least not for Android. I tested with react-native 0.60.5 and react 16.8.6

> System:
>     OS: Windows 10
>     CPU: (4) x64 Intel(R) Core(TM) i7-2620M CPU @ 2.70GHz
>     Memory: 3.39 GB / 11.95 GB
>   Binaries:
>     Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
>     Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
>     npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
>     Watchman: 4.9.4 - C:\Users\Dipo\bin\watchman\watchman.EXE
>   SDKs:
>     Android SDK:
>       Android NDK: 20.0.5594570
>   IDEs:
>     Android Studio: Version  3.5.0.0 AI-191.8026.42.35.5791312
> 

The app stops immediately after it's launched and the following error is logged in the adb logcat.

> java.lang.RuntimeException: Unable to instantiate application com.example.app.MainApplication: java.lang.ClassNotFoundException: Didn't find class "com.example.app.MainApplication" on path: DexPathList[[zip file "/data/app/com.example.app-1/base.apk"],nativeLibraryDirectories=[/data/app/com.example.app-1/lib/x86, /data/app/com.example.app-1/base.apk!/lib/x86, /vendor/lib, /system/lib]]

However, after doing the manual linking (as described [here](https://github.com/joltup/rn-fetch-blob/wiki/Manually-Link-Package#index)), it works.

**Comment by judygab** [comment]
> Auto linking could mean 2 things.
> Before react-native 0.60, auto-linking used to mean running the "react-native link" command.
> After react-native 0.60 auto-linking now means not having to run the "react-native link" command.
> 
> I can confirm that the module does not support the auto-linking feature that comes with react-native 0.60. At least not for Android. I tested with react-native 0.60.5 and react 16.8.6
> 
> > System:
> > OS: Windows 10
> > CPU: (4) x64 Intel(R) Core(TM) i7-2620M CPU @ 2.70GHz
> > Memory: 3.39 GB / 11.95 GB
> > Binaries:
> > Node: 12.16.1 - C:\Program Files\nodejs\node.EXE
> > Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
> > npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
> > Watchman: 4.9.4 - C:\Users\Dipo\bin\watchman\watchman.EXE
> > SDKs:
> > Android SDK:
> > Android NDK: 20.0.5594570
> > IDEs:
> > Android Studio: Version  3.5.0.0 AI-191.8026.42.35.5791312
> 
> The app stops immediately after it's launched and the following error is logged in the adb logcat.
> 
> > java.lang.RuntimeException: Unable to instantiate application com.example.app.MainApplication: java.lang.ClassNotFoundException: Didn't find class "com.example.app.MainApplication" on path: DexPathList[[zip file "/data/app/com.example.app-1/base.apk"],nativeLibraryDirectories=[/data/app/com.example.app-1/lib/x86, /data/app/com.example.app-1/base.apk!/lib/x86, /vendor/lib, /system/lib]]
> 
> However, after doing the manual linking (as described [here](https://github.com/joltup/rn-fetch-blob/wiki/Manually-Link-Package#index)), it works.
> 
> PS
> I got duplicate module error so I had to undo the MainApplication.java setup.
If I'm using RN 0.6+, do I have to manually link libraries than or I can just do react-native link?

**Comment by sam9010** [comment]
do you mean?
after run this code
npm install --save rn-fetch-blob
what do i do?
